code/modules/projectiles/ammo_datums.dm

```diff
@@ -172,7 +172,7 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 
 
 	//This is sort of a workaround for now. There are better ways of doing this ~N.
-/datum/ammo/proc/stun_living(mob/living/target, obj/projectile/proj) //Taser proc to stun folks.
+/datum/ammo/proc/stun_living(mob/living/target, obj/projectile/proj) //Stuns people.
```

Comment:
just delete this entire proc

Reply:
the effect should be applied to everyone the same and use the standard procs instead of this snowflake one